packages/paste-website/src/pages/foundations/data-visualization/for-engineers/index.mdx

```diff
@@ -0,0 +1,148 @@
+export const meta = {
+  title: "Data Visualization",
```

Comment:
todo: Change the title to match whatever the sidebar item is called, like "Getting started" or "Setting up charts", etc.